test/store_test.js

```diff
@@ -61,6 +61,34 @@ describe("Store", () => {
   });
 
 
+  describe("#update()", () => {
+
+    it("update() replaces with Kinto record and emits change", (done) => {
```

Comment:
Nit: uneecessary `update()` mention as we're already in a section mentioning it.